.github/workflows/release.yaml

```diff
@@ -45,7 +35,9 @@ jobs:
         run: |
           pip install hatch
 
-      - name: Build and Distribute
+      - name: Build Package
         run: |
           hatch build
-          hatch publish
+
+      - name: Upload to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Wow nice how simple this fix was

```
This action allows you to upload your [Python distribution packages](https://packaging.python.org/glossary/#term-Distribution-Package) in the dist/ directory to PyPI
```

Looking at the action. Awesome I think this should work.